sdk/apis/kubermatic/v1/configuration.go

```diff
@@ -229,6 +229,12 @@ type KubermaticSeedControllerConfiguration struct {
 	// DisabledCollectors contains a list of metrics collectors that should be disabled.
 	// Acceptable values are "Addon", "Cluster", "ClusterBackup", "Project", and "None".
 	DisabledCollectors []MetricsCollector `json:"disabledCollectors,omitempty"`
+	// BackupInterval defines the time duration between consecutive etcd backups.
+	// Must be a valid time.Duration string format. Only takes effect when backup scheduling is enabled.
+	BackupInterval string `json:"backupInterval,omitempty"`
```

Comment:
[metav1.Duration](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration) seems to be the better type for the job as it's already will be validated and parsed accordingly.